I'm failing to build `acl2` with this:

```
   | Using default SMT-trusted-cp...
   | ; SMT solver: `python /tmp/py_file/smtlink.NPhwN`: 0.19 sec, 0 bytes
   | 
   | 
   | HARD ACL2 ERROR in SMT-RUN=>SMT-INTERPRET:  Z3 failure: ("Traceback (most recent call last):"
   |                                                          "File \"/tmp/py_file/smtlink.NPhwN\", line 4, in <module>"
   |                                                          "from ACL2_to_Z3 import *"
   |                                                          "File \"/nix/store/369a2pdq1idjyig0ylad1chvd42r6985-acl2-8.5/share/acl2/books/projects/smtlink/z3_interface/ACL2_to_Z3.py\", line 9, in <module>"
   |                                                          "from z3 import *"
   |                                                          "ModuleNotFoundError: No module named 'z3'")
   | 
   | 
   | 
   | 
   | ACL2 Error in ( DEFTHM POLY-INEQ-EXAMPLE ...):  Evaluation aborted.
   | To debug see :DOC print-gv, see :DOC trace, and see :DOC wet.
```

Failure suggests it's python related, aarch64-linux.

--- a/pkgs/development/interpreters/acl2/default.nix
+++ b/pkgs/development/interpreters/acl2/default.nix
@@ -1,6 +1,6 @@
 { lib, stdenv, callPackage, fetchFromGitHub, runCommandLocal, makeWrapper, substituteAll
 , sbcl, bash, which, perl, hostname
-, openssl, glucose, minisat, abc-verifier, z3, python2
+, openssl, glucose, minisat, abc-verifier, z3, python3
 , certifyBooks ? true
 } @ args:
 
@@ -50,7 +50,7 @@ in stdenv.mkDerivation rec {
     which perl hostname
     # Some of the books require one or more of these external tools:
     glucose minisat abc-verifier libipasir
-    z3 (python2.withPackages (ps: [ ps.z3 ]))
+    z3 (python3.withPackages (ps: [ ps.z3 ]))
   ];
 
   # NOTE: Parallel building can be memory-intensive depending on the number of

